src/providers/wallet-provider.tsx

```
@@ -1,39 +1,75 @@
+    console.log("All available connectors:", 
+      allConnectors.map(c => ({ id: c.id, name: c.name }))
     );
-}
+  }, []);
+
+  const provider = jsonRpcProvider({
+    rpc: (chain: Chain) => {
+      switch (chain) {
+        case mainnet:
+          return { nodeUrl: "https://api.cartridge.gg/x/starknet/mainnet" };
+        case sepolia:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)